Images/Moon Font/License.txt

```diff
@@ -0,0 +1,8 @@
+FOR PERSONAL USE ONLY.

```

Comment:
Maybe. I suppose it may not qualify as "personal use", but it also doesn't qualify as "commercial use". Let's just be awesome and reach out to the guy and pay him anyway.